.github/workflows/ci.yaml

```diff
@@ -73,5 +73,8 @@ jobs:
           password: ${{secrets.GITHUB_TOKEN}}
       - name: Build and push container image
         run: |
-          docker build . -f docker/Dockerfile --tag ghcr.io/holo-routing/holo:latest
+          docker build . -f docker/Dockerfile --tag ghcr.io/holo-routing/holo:latest ghcr.io/holo-routing/holo:0.6.0
```

Comment:
This would cause all CI runs on `master` to update the `v0.6.0` image as well.

It might be easier to push release versions manually. If we want to automate this via CI, we'll need to check if the git commit has an associated tag. Then, if it does, the CI should push a docker image tagged with that version. If not, the CI should default to using the `holo:latest` tag.